aws-rds-dbparametergroup/src/main/java/software/amazon/rds/dbparametergroup/Translator.java

```diff
@@ -135,4 +131,11 @@ static ResourceModel translateFromDBParameterGroup(final DBParameterGroup dbPara
                         .build())
                 .collect(Collectors.toList());
     }
+
+    private static String getParameterApplyMethod(Parameter parameter) {
+        String applyMethod = ApplyMethod.PENDING_REBOOT.toString();
+        if (parameter.applyType().equalsIgnoreCase(ParameterType.Dynamic.toString()))   // If the parameter is DYNAMIC, we can apply now
+            applyMethod = ApplyMethod.IMMEDIATE.toString();
+        return applyMethod;
+    }
```

Comment:
nit: this could be slightly re-shaped:
```java
private static ApplyMethod getPatameterApplyMethod(final Parameter parameter) {
    if (ParameterType.Dynamic.toString().equalsIgnoreCase(parameter.applyType()) {
        return ApplyMethod.IMMEDIATE;
    }
    return ApplyMethod.PENDING_REBOOT;
}
```

We can also define a bit of syntax sugar in ParameterType called `public bool equalsString(final String parameterType)`

Note: this is a purely cosmetic suggestion.

Reply:
Done.